storage_controller/src/service.rs

```diff
@@ -195,6 +198,7 @@ pub(crate) enum LeadershipStatus {
 }
 
 pub const RECONCILER_CONCURRENCY_DEFAULT: usize = 128;
+pub const PRIORITY_RECONCILER_CONCURRENCY_DEFAULT: usize = 256;
```

Comment:
Perhaps in a follow up. Could we make these cli params? Would've been useful last night.